test/unit/test_actions.py

```diff
@@ -4,6 +4,7 @@
 
 
 def test_create_action(gl: ExperimentalApi):
+    gl.delete_all_rules()
```

Comment:
Can we add a comment here mentioning that we are deleting rules first just so previous runs don't affect results from the current test run?